lms/envs/common.py

```diff
@@ -1861,6 +1861,12 @@
     'provider.oauth2',
     'oauth2_provider',
 
+    # We don't use this directly (since we use OAuth2), but we need to install it anyway.
+    # When a user is deleted, Django queries all tables with a FK to the auth_user table,
+    # and since django-rest-framework-oauth imports this, it will try to access tables
+    # defined by oauth_provider.  If those tables don't exist, an error can occur.
+    'oauth_provider',
```

Comment:
These are the migrations: https://bitbucket.org/david/django-oauth-plus/src/448b6729ac6aa1448c323495ca42c0590a2da225/oauth_provider/south_migrations/?at=default

The migrations look sane to me.  Basically, there are four tables that get created (and which our application code won't be using).  If we roll-back, the tables will be deleted.